azure-samples/src/main/java/com/microsoft/azure/management/documentdb/samples/DocumentDBWithEventualConsistency.java

```diff
@@ -0,0 +1,166 @@
+import com.microsoft.azure.management.samples.Utils;
+import com.microsoft.rest.LogLevel;
+
+import java.io.File;
+
+/**
+ * Azure DocumentDB sample for high availability -
+ *  - Create a documentdb configured with eventual consistency
+ *  - Get the credentials for the documentdb
+ *  - add collection to the docuemntdb
+ *  - Delete the documentdb.
+ */
+public final class DocumentDBWithEventualConsistency {
+    static final String DATABASE_ID = "TestDB";
+    static final String COLLECTION_ID = "TestCollection";
+
+    /**
+     * Main function which runs the actual sample.
+     * @param azure instance of the azure client
+     * @param clientId client id
+     * @return true if sample runs successfully
+     */
+    public static boolean runSample(Azure azure, String clientId) {
+        final String docDBName = SdkContext.randomResourceName("docDb", 10);
+        final String rgName = SdkContext.randomResourceName("rgNEMV", 24);
+
+        try {
+            //============================================================
+            // Create a documentdb.
+
+            System.out.println("Creating a documentdb...");
+            DocumentDBAccount documentDBAccount = azure.documentDBs().define(docDBName)
+                    .withRegion(Region.US_WEST)
+                    .withNewResourceGroup(rgName)
+                    .withKind(DatabaseAccountKind.GLOBAL_DOCUMENT_DB)
+                    .withEventualConsistency()
+                    .withWriteReplication(Region.US_EAST)
+                    .withReadReplication(Region.US_CENTRAL)
+                    .create();
+
+            System.out.println("Created documentdb");
+            Utils.print(documentDBAccount);
+
+            //============================================================
+            // Get credentials for the documentdb.
+
+            System.out.println("Get credentials for the documentdb");
+            DatabaseAccountListKeysResult databaseAccountListKeysResult = documentDBAccount.listKeys();
+            String masterKey = databaseAccountListKeysResult.primaryMasterKey();
+            String endPoint = documentDBAccount.documentEndpoint();
+
+            //============================================================
+            // Connect to documentdb and add a collection
+
+            System.out.println("Connecting and adding collection");
+            createDBAndAddCollection(masterKey, endPoint);
+
+            //============================================================
+            // Delete documentdb
+            System.out.println("Deleting the docuemntdb");
+            azure.documentDBs().deleteById(documentDBAccount.id());
+            System.out.println("Deleted the documentdb");
+
+            return true;
+        } catch (Exception e) {
+            System.err.println(e.getMessage());
+        } finally {
+            try {
+                System.out.println("Deleting Resource Group: " + rgName);
```

Comment:
Capitalization: no need to capitalize "Resource Group" in a user message, as it is not a proper name